aea/helpers/profiling.py

```diff
@@ -47,26 +47,22 @@
 
 if platform.system() == "Windows":  # pragma: nocover
 
+    import win32process  # type: ignore  # pylint: disable=import-error,import-outside-toplevel
```

Comment:
Why did you move it back out?

Reply:
So this was wrong, the failure was not caused by this logic. It was a situation similar to missing `_strptime` module yesterday. The `win32process` import was not the issue, the issue was of missing `win32timezone` reference while importing `win32process` which was fixed in the latest commit